src/storage/heap_file.c

```diff
@@ -14520,7 +14520,24 @@ heap_dump (THREAD_ENTRY * thread_p, FILE * fp, HFID * hfid, bool dump_records)
 void
 heap_dump_heap_file (THREAD_ENTRY * thread_p, FILE * fp, bool dump_records, const char *class_name)
 {
-  /* TODO: Fetch HFID of class which corresponds to class_name, and dump heap file by HFID. Will be handled at CBRD-25313 and CBRD-25314. */
+  int error_code = NO_ERROR;
+  OID class_oid;
+  LC_FIND_CLASSNAME status;
+  HFID hfid;
+
+  status = xlocator_find_class_oid (thread_p, class_name, &class_oid, S_LOCK);
+  if (status != LC_CLASSNAME_EXIST)
+    {
+      return;
```

Comment:
We need to handle the exceptional cases. 
1. We can define error code (`error_code.h`), set error code here (`er_set`), and then return the error code to the user.
2. Or, we can write the error message into the file (at fp)

Reply:
When the class which has `class_name` doesn't exists, it was decided to print any message (Only print Head and Tail message of dump). And it was due to coherence with other utility functions.

However, for error_code for `heap_hfid_cache_get` function, we can call assertion to check HFID exists.

Reply:
OK, I agree